ff-parser/src/main/java/org/uniprot/core/flatfile/parser/impl/cc/cclineobject/CcLineObject.java

```diff
@@ -0,0 +1,116 @@
+package org.uniprot.core.flatfile.parser.impl.cc.cclineobject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.uniprot.core.flatfile.parser.impl.EvidenceInfo;
+import org.uniprot.core.flatfile.parser.impl.HasEvidenceInfo;
+
+/** User: wudong, Date: 03/09/13, Time: 16:35 */
+public class CcLineObject implements HasEvidenceInfo {
+
+    private List<CC> ccs = new ArrayList<>();
+    private EvidenceInfo evidenceInfo = new EvidenceInfo();
+
+    public List<CC> getCcs() {
+        return ccs;
+    }
+
+    public void setCcs(List<CC> ccs) {
+        this.ccs = ccs;
+    }
+
+    @Override
+    public EvidenceInfo getEvidenceInfo() {
+        return evidenceInfo;
+    }
+
+    public void setEvidenceInfo(EvidenceInfo evidenceInfo) {
+        this.evidenceInfo = evidenceInfo;
+    }
+
+    public enum AlternativeNameSequenceEnum {
```

Comment:
Maybe these enums could go inside the class that is using it... for example, AlternativeNameSequenceEnum could go inside AlternativeProductName.. is it possible?